posix-configs/SITL/init/ekf2/rover

```diff
@@ -50,23 +63,24 @@ pwm_out_sim mode_pwm
 sleep 1
 sensors start
 commander start
+land_detector start rover
 navigator start
 ekf2 start
-fw_pos_control_l1 start
-fw_att_control start
-land_detector start rover
+gnd_pos_control start
+gnd_att_control start
 mixer load /dev/pwm_output0 ROMFS/sitl/mixers/rover_sitl.main.mix
-mavlink start -u 14556 -r 2000000
-mavlink start -u 14557 -r 2000000 -m onboard -o 14540
+mavlink start -u 14556 -r 4000000
+mavlink start -u 14557 -r 4000000 -m onboard -o 14540
 mavlink stream -r 80 -s POSITION_TARGET_LOCAL_NED -u 14556
 mavlink stream -r 80 -s LOCAL_POSITION_NED -u 14556
 mavlink stream -r 80 -s GLOBAL_POSITION_INT -u 14556
 mavlink stream -r 80 -s ATTITUDE -u 14556
 mavlink stream -r 80 -s ATTITUDE_QUATERNION -u 14556
 mavlink stream -r 80 -s ATTITUDE_TARGET -u 14556
+mavlink stream -r 50 -s SERVO_OUTPUT_RAW_0 -u 14556
 mavlink stream -r 20 -s RC_CHANNELS -u 14556
 mavlink stream -r 250 -s HIGHRES_IMU -u 14556
 mavlink stream -r 10 -s OPTICAL_FLOW_RAD -u 14556
-logger start -e -t
```

Comment:
This was done in commit [ab88a20](https://github.com/PX4/Firmware/pull/7139/commits/ab88a205509ac061d2742e0f88bdb54e8371a468)